langchain4j-azure-ai-search/src/test/java/dev/langchain4j/store/embedding/azure/search/AzureAiSearchContentRetrieverTestIT.java

```diff
@@ -0,0 +1,252 @@
+package dev.langchain4j.store.embedding.azure.search;
+
+import dev.langchain4j.data.embedding.Embedding;
+import dev.langchain4j.data.segment.TextSegment;
+import dev.langchain4j.model.embedding.AllMiniLmL6V2QuantizedEmbeddingModel;
+import dev.langchain4j.model.embedding.EmbeddingModel;
+import dev.langchain4j.rag.content.Content;
+import dev.langchain4j.rag.query.Query;
+import dev.langchain4j.store.embedding.*;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.EnabledIfEnvironmentVariable;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+
+import static java.util.Arrays.asList;
+import static org.assertj.core.api.Assertions.assertThat;
+
+@EnabledIfEnvironmentVariable(named = "AZURE_SEARCH_ENDPOINT", matches = ".+")
+public class AzureAiSearchContentRetrieverTestIT extends EmbeddingStoreIT {
```

Comment:
Could you please add a tests that verifies that `FULL_TEXT` can be used on it's own, without embeddings (and `EmbeddingModel`)?

Reply:
Oh, there's a bug here, you currently need to have an embedding. I'm fixing this.

Reply:
Done